sycl/doc/GettingStartedAlveo.md

```diff
@@ -308,6 +308,7 @@ cmake -DCMAKE_BUILD_TYPE=Release \
   -DLLVM_ENABLE_PROJECTS="clang" \
   -DLLVM_EXTERNAL_SYCL_SOURCE_DIR=$SYCL_HOME/sycl \
   -DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=$SYCL_HOME/llvm-spirv \
+  -DLLVM_BUILD_LLVM_DYLIB:BOOL=ON -DLLVM_LINK_LLVM_DYLIB:BOOL=ON \
```

Comment:
Note: `LLVM_LINK_LLVM_DYLIB` automatically sets `LLVM_BUILD_LLVM_DYLIB` to `ON`. From [documentation](https://llvm.org/docs/CMake.html):

> LLVM_LINK_LLVM_DYLIB:BOOL
>      If enabled, tools will be linked with the libLLVM shared library. Defaults to OFF. Setting LLVM_LINK_LLVM_DYLIB to ON also sets LLVM_BUILD_LLVM_DYLIB to ON.

Reply:
Very good point! I am updating the documentation.